framework/organisms/nav-tabs/_nav-tabs.scss

```diff
@@ -16,12 +16,12 @@ ul.nav-tabs,
   margin-bottom: 2em !important;
   border-bottom: 2px solid $grey;
 
-  & > li {
+  > li {
     margin-bottom: -2px;
 
     &.active {
-      & > a {
-        border-bottom: 2px solid #fff;
+      > a {
+        border-bottom: 2px solid $white;
```

Comment:
I honestly don't see value in having a variable for white and black, as they are so pure, but not a stopper as well. Just no clear value.